cmapPy/math/tests/test_fast_cov.py

```diff
@@ -0,0 +1,243 @@
+import unittest
+import logging
+import cmapPy.pandasGEXpress.setup_GCToo_logger as setup_logger
+import cmapPy.math.fast_cov as fast_cov
+import numpy
+
+
+logger = logging.getLogger(setup_logger.LOGGER_NAME)
+
+# Some notes on testing conventions (more in cuppers convention doc):
```

Comment:
I think the "Notes on testing conventions" comment can be deleted? Assuming you just used the template

Reply:
Yes, good catch.  Just left it there out of habit :)